lib/features/email/presentation/extensions/calendar_event_extension.dart

```diff
@@ -175,4 +177,42 @@ extension CalendarEventExtension on CalendarEvent {
       return Colors.transparent;
     }
   }
+
+  DateTime? get localStartDate => startUtcDate?.value.toLocal();
+
+  String get monthStartDateAsString {
```

Comment:
is return type as `String?` better or `String`, wdyt ?

Reply:
We want this value to be non-null.